src/app/app.component.html

```diff
@@ -1,13 +1,61 @@
-<ng-container *ngFor="let graph of graphData; let i = index">
+<ng-container *ngIf="!isEditing; else editor">
+  <ng-container *ngFor="let graph of graphs; let i = index">
+    <div class="chart-container">
+      <button
+        class="edit"
+        (click)="startEditing(i)"
+      >
+        Edit
+      </button>
+      <multi-line-chart
+        yAxisLabel="Number of confirmed cases"
+        xAxisLabel="Days since {{ graph.definition.cutoff }} cases"
+        [data]="graph.data"
+        [animate]="graph.definition.animate"
+        [xAxisBounds]="graph.definition.x_axis_bounds"
+        [yAxisBounds]="graph.definition.y_axis_bounds"
+        [yAxisScale]="graph.definition.y_axis_scale"
+      >
+      </multi-line-chart>
+    </div>
+  </ng-container>
+</ng-container>
+<ng-template #editor>
+  <div>
+    <button
+      class="back"
+      (click)="stopEditing()"
+    >
+      Back
+    </button>
+  </div>
   <auto-suggest
-    [index]="i"
-    [graphDefinitions]="graphDefinitions"
+    [index]="editingIndex"
+    [graphDefinitions]="allGraphDefinitions"
   ></auto-suggest>
   <multi-line-chart
```

Comment:
TODO: Some code duplication now -- <multi-line-chart> is in app.component.html twice with same params repeated. p annoying, should fix before merging